insighthub-frontend/src/pages/Dashboard.tsx

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
 import { Container, Typography, Box, Button } from '@mui/material';
 import { useNavigate } from 'react-router-dom';
+import { config } from '../config';
 
 const Dashboard: React.FC = () => {
+
   const navigate = useNavigate();
 
   const handleLogout = () => {
-    // Clear any authentication tokens or user data here
-    navigate('/login');
+    // Add your logout logic here
```

Comment:
Reminder: We need to also call the API of `/auth/logout` here.

https://github.com/Lina0Elman/InsightHub/issues/3#issuecomment-2679488736